blackbox/bootstrap.sh

```diff
@@ -22,7 +22,9 @@
 
 DIR=$(dirname "$0")
 
-if hash python3.7 2> /dev/null; then
+if hash python3.9 2> /dev/null; then
+    PYTHON=python3.9
+elif hash python3.7 2> /dev/null; then
```

Comment:
Do we still need this?

Reply:
Probably yes, it might work around some problem where `python3` points/pointed to an older Python3 (e.g. `python3.5`) and only `python3.7` will be fine to invoke the stack. On those environments, there might not be a `python3.9` yet.

I can't tell if we _still_ need it.

Reply:
I'd remove it and re-add if really needed.

Reply:
Hi. All right, I just amended 899460d according to your suggestions.